src/AppInstallerCLICore/Workflows/PortableFlow.cpp

```diff
@@ -303,14 +455,66 @@ namespace AppInstaller::CLI::Workflow
             }
 
             std::filesystem::create_symlink(targetFullPath, symlinkFullPath);
-            AICLI_LOG(CLI, Info, << "Symlink created at: " << symlinkFullPath);
-
+            AICLI_LOG(CLI, Info, << "Symlink created at: " << symlinkFullPath);
+            uninstallEntry.SetValue(PortableValueName::PortableSymlinkFullPath, symlinkFullPath.wstring());
+
             if (AddToPathRegistry(context))
             {
                 context.Reporter.Warn() << Resource::String::ModifiedPathRequiresShellRestart << std::endl;
             }
         }
 
+        void RemovePortableSymlink(Execution::Context& context)
+        {
+            PortableARPEntry uninstallEntry = GetPortableARPEntryForUninstall(context);
+            const auto& symlinkPath = uninstallEntry[PortableValueName::PortableSymlinkFullPath];
+            if (symlinkPath.has_value())
+            {
+                const std::filesystem::path& symlinkPathValue = symlinkPath.value().GetValue<Value::Type::UTF16String>();
+
+                if (!std::filesystem::remove(symlinkPathValue))
+                {
+                    AICLI_LOG(CLI, Info, << "Portable symlink not found; Unable to delete portable symlink: " << symlinkPathValue);
+                }
+            }
+            else
+            {
+                AICLI_LOG(CLI, Info, << "The registry value for [SymlinkFullPath] does not exist");
+            }
+
+            std::string_view installedScope;
+            if (context.Args.Contains(Execution::Args::Type::InstallScope))
+            {
+                installedScope = context.Args.GetArg(Execution::Args::Type::InstallScope);
+            }
+            else
+            {
+                installedScope = context.Get<Execution::Data::InstalledPackageVersion>()->GetMetadata()[Repository::PackageVersionMetadata::InstalledScope];
+            }
+
+            if (std::filesystem::is_empty(GetPortableLinksLocation(ConvertToScopeEnum(installedScope))))
+            {
+                RemoveFromPathRegistry(context);
+            }
+
+            uninstallEntry.Delete();
```

Comment:
Please intentionally remove the registry key on its own rather than hiding it inside the function for removing the symlink.

Reply:
Created a separate task for removing the registry key.